### DISCUSSION TITLE
Custom ConnectModalIntro

### DISCUSSION THREAD

**Discussion by niratodev** [rainbow-me/rainbowkit#582] [Ideas]
Hi, great library. Over here at LooksRare we are considering migrating to this wallet. One thing holding us back is we want to add a custom `ConnectModalIntro` component.

Do you guys have any plans on this feature? I might be able to squeeze in some time to submit a PR myself but didn't want to impede on any current development work.

**Comment by Yuripetusko** [comment]
I second this. Especially since we are using it on other EVM chains that are not Ethereum, so this intro mentioning Ethereum is confusing to users.
